conditional-signing/nodejs/src/index.ts

```diff
@@ -0,0 +1,185 @@
+      });
+    console.log("✅ Capacity Delegation Auth Sig created");
+
+    console.log("🔄 Executing Lit Action...");
+    const litActionSignatures = await litNodeClient.executeJs({
+      sessionSigs: await litNodeClient.getSessionSigs({
+        chain: CHAIN_TO_CHECK_CONDITION_ON,
+        capabilityAuthSigs: [capacityDelegationAuthSig],
+        expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+        resourceAbilityRequests: [
+          {
+            resource: new LitPKPResource("*"),
+            ability: LitAbility.PKPSigning,
+          },
+          {
+            resource: new LitActionResource("*"),
+            ability: LitAbility.LitActionExecution,
+          },
+        ],
+        authNeededCallback: async ({
+          resourceAbilityRequests,
+          expiration,
+          uri,
+        }) => {
+          const toSign = await createSiweMessageWithRecaps({
+            uri: uri!,
+            expiration: expiration!, // 24 hours
+            resources: resourceAbilityRequests!,
+            walletAddress: await ethersWallet.getAddress(),
+            nonce: await litNodeClient.getLatestBlockhash(),
+            litNodeClient: litNodeClient,
+          });
+
+          return await generateAuthSig({
+            signer: ethersWallet,
+            toSign,
+          });
+        },
+      }),
+      code: litActionCode,
+      jsParams: {
+        conditions: [
+          {
+            conditionType: "evmBasic",
+            contractAddress: "",
+            standardContractType: "",
+            chain: CHAIN_TO_CHECK_CONDITION_ON,
+            method: "eth_getBalance",
+            parameters: [":userAddress", "latest"],
+            returnValueTest: {
+              comparator: ">=",
+              value: "1",
+            },
+          },
+        ],
+        authSig: await (async () => {
+          const toSign = await createSiweMessageWithRecaps({
+            uri: "http://localhost",
+            expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+            walletAddress: await ethersWallet.getAddress(),
+            nonce: await litNodeClient.getLatestBlockhash(),
+            resources: [
+              {
+                resource: new LitPKPResource("*"),
+                ability: LitAbility.PKPSigning,
+              },
+              {
+                resource: new LitActionResource("*"),
+                ability: LitAbility.LitActionExecution,
+              },
+            ],
+            litNodeClient: litNodeClient,
+          });
+          return await generateAuthSig({
+            signer: ethersWallet,
+            toSign,
+          });
+        })(),
+        chain: CHAIN_TO_CHECK_CONDITION_ON,
+        dataToSign: ethers.utils.arrayify(ethers.utils.keccak256([1, 2, 3, 4, 5])),
+        publicKey: pkpInfo.publicKey,
+      },
+    });
+    console.log("✅ Lit Action executed successfully");
+    console.log(litActionSignatures);
```

Comment:
```suggestion
```

I don't think we need to log these since they're large and displace the rest of the logs significantly